File: etl/models/bem_candidato.py
from sqlalchemy import Column, BigInteger, String, Integer, ForeignKey, Numeric, PrimaryKeyConstraint
from .base import Base

class BemCandidato(Base):
    __tablename__ = "bem_candidato"

    sq_candidato = Column(BigInteger, ForeignKey("candidatura.sq_candidato"), nullable=False)
    nr_ordem_bem = Column(Integer, nullable=False)
    cd_tipo_bem = Column(Integer)
    ds_tipo_bem = Column(String)
    ds_bem = Column(String)
    vr_bem = Column(Numeric(15, 2))

    __table_args__ = (
        PrimaryKeyConstraint("sq_candidato", "nr_ordem_bem"),
    )

File: src/backend/VoteBem/Entities/BemCandidato.cs
namespace VoteBem.Entities
{
    public class BemCandidato
    {
        public long SqCandidato { get; private set; }
        public int NrOrdemBem { get; private set; }
        public int? CdTipoBem { get; private set; }
        public string? DsTipoBem { get; private set; }
        public string? DsBem { get; private set; }
        public decimal? VrBem { get; private set; }

        public Candidatura Candidatura { get; private set; } = null!;

        protected BemCandidato() { }
    }
}
